coordinator/pom.xml

```diff
@@ -1,19 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <!-- Artifact props -->
   <groupId>io.stargate</groupId>
   <artifactId>stargate</artifactId>
+  <name>Stargate - Coordinator Parent pom</name>
   <version>2.0.10-SNAPSHOT</version>
   <packaging>pom</packaging>
-  <name>${project.groupId}:${project.artifactId}</name>
-  <description>Core modules for Stargate</description>
-  <url>http://github.com/stargate/stargate</url>
+  <!-- Description -->
+  <description>Stargate - An open source data API gateway</description>
+  <url>https://github.com/stargate/stargate</url>
+  <!-- Dev props -->
+  <developers>
+    <developer>
+      <id>stargate</id>
+      <name>The Stargate Authors</name>
+      <email>maintainers@stargate.io</email>
+      <organization>Stargate</organization>
+      <organizationUrl>https://stargate.io</organizationUrl>
+    </developer>
+  </developers>
+  <!-- License -->
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <!-- Source code control management -->
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
+    <url>https://github.com/stargate/stargate</url>
+    <connection>scm:git:git://github.com/stargate/stargate.git</connection>
+    <developerConnection>scm:git:git://github.com/stargate/stargate.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+  <!-- Issue management -->
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/stargate/stargate/issues/</url>
+  </issueManagement>
+  <!-- Shared properties -->
   <properties>
+    <!-- Compiler -->
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-
-    <osgi.bundle.version>${project.version}</osgi.bundle.version>
+    <!-- Encoding -->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <!-- OSGi -->
+    <osgi.bundle.version>${project.version}</osgi.bundle.version>
+    <!-- Release -->
     <project.scm.id>github</project.scm.id>
     <doclint>none</doclint>
     <driver.version>4.15.0</driver.version>
```

Comment:
the line below is very interesting, the XML formatting for poms is disabled here..  guess we should enable and reformat? The reason I added those inline comments was to separate a bit definitions in the pom, as new lines get killed by the formatter..

Reply:
That is interesting since I think I have gotten a CI fail on not having run xml formatting -- so it is enabled for some xml files (but not pom.xml's?). I wonder if there was a problem with indentation resolved by disabling, but if so, what? (and why no comment to indicate the reason(s))

Reply:
No idea. Not even sure how did this end up there.. But I checked the formatting check will not complain, already bunch of files are out of format. Remove and re-format?

Reply:
Sounds good to me. We can disable again if there are issues.

Reply:
Reformatted and removed that prop..